### PR TITLE
[SCB-590] Update the validator version

### PR DESCRIPTION
**Pull request by acsukesh**
Follow this checklist to help us incorporate your contribution quickly and easily:

 - [x] Make sure there is a [JIRA issue](https://issues.apache.org/jira/browse/SCB) filed for the change (usually before you start working on it).  Trivial changes like typos do not require a JIRA issue.  Your pull request should address just this issue, without pulling in other changes.
 - [x] Each commit in the pull request should have a meaningful subject line and body.
 - [x] Format the pull request title like `[SCB-XXX] Fixes bug in ApproximateQuantiles`, where you replace `SCB-XXX` with the appropriate JIRA issue.
 - [x] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
 - [x] Run `mvn clean install` to make sure basic checks pass. A more thorough check will be performed on your pull request automatically.
 - [x] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

---

### CODE REVIEW

File: java-chassis-dependencies/pom.xml [hunk 1]
```diff
@@ -324,7 +324,7 @@
       <dependency>
         <groupId>org.hibernate</groupId>
         <artifactId>hibernate-validator-annotation-processor</artifactId>
-        <version>5.2.2.Final</version>
+        <version>6.0.2.Final</version>
```
Comment (wujimin) [inline java-chassis-dependencies/pom.xml]: license file need to be updated too.

Comment (WillemJiang) [reply java-chassis-dependencies/pom.xml]: Please update the version in the https://github.com/apache/incubator-servicecomb-java-chassis/blob/master/java-chassis-distribution/src/release/LICENSE#L349

Comment (acsukesh) [reply java-chassis-dependencies/pom.xml]: Fixed

Comment (WillemJiang) [reply java-chassis-dependencies/pom.xml]: It looks like the annotation processor is option one,  it's just for the IDE validation.  Do we need to ship it?

**Comment by yangbor** [comment]
@WillemJiang The hibernate-validator-annotation-processor is not shipped with the binary release(I chechked that the jar is not packed in to the binary release). hibernate-validator is shipped and mentioned in the license file.

As for other jars missing in the license file, I'll check and add them after the feature freeze.

**Pull request merged**